### ISSUE TITLE
Training schedule for segmentation net

### ISSUE THREAD

**Issue by arnike** [jiwoon-ahn/psa#24]
Hi Jiwoon,

when training the segmentation net on the final masks, what schedule did you use (e.g. learning rate/decay, num of iterations/epochs, frozen/learned BN)? My guess is that you might need early stopping to avoid overfitting to the errors in the masks used for pseudo-supervision.

Thanks,
Nikita

**Comment by liuyue718** [comment]
Hi, I would like to ask how your --voc12_root and --weights are assigned, why did I write the local position of the dataset wrong?
